index.html

```diff
@@ -52,115 +55,8 @@ <h1 class="section-heading" id="about-heading">About</h1>
             <hr class="body-border">
             <div class="schedule" id="schedule">
                 <h1 class="section-heading" id="schedule-heading">Schedule</h1>
-                <h1 class="sponsors-label sponsors-h2" id="schedule-header-main">Main Agenda</h1>
```

Comment:
Let's comment this out instead of deleting so that we can easily come back to it once we do have our schedule ready.